sentry-android-core/src/main/java/io/sentry/android/core/AndroidOptionsInitializer.java

```diff
@@ -65,6 +67,33 @@ static void init(
     // because sentry-native move files around and we don't want to watch that.
     options.addIntegration(new NdkIntegration());
     options.addIntegration(EnvelopeFileObserverIntegration.getOutboxFileObserver(envelopeReader));
+
+    // Send cached envelopes from outbox path
+    // this should be executed after NdkIntegration because sentry-native move files on init.
+    // and we'd like to send them right away
+    options.addIntegration(
+        new SendCachedEventFireAndForgetIntegration(
```

Comment:
One improvement is to pull this code into a its own class and adding here `new Integration()`
This would simplify this method a bit

Reply:
yep, could be done for the others too then